dashboard/app/views/layouts/_header.html.haml

```diff
@@ -34,10 +34,6 @@
               - else
                 = link_to(image_tag('codeorg-studio-logo.png'), '/')
             %span.project_info
```

Comment:
I think we need this element here for other stuff (e.g. project name and Remix button): https://github.com/code-dot-org/code-dot-org/blob/1b36ed055372c54b67ba1cec9299bfc37a80b8e7/dashboard/app/assets/javascripts/header.js#L300